src/libConsensus/ConsensusLeader.cpp

```diff
@@ -107,13 +108,12 @@ void ConsensusLeader::GenerateConsensusSubsets() {
       peersWhoCommitted.push_back(index);
     }
   }
-  // Generate NUM_CONSENSUS_SUBSETS lists (= subsets of peersWhoCommitted)
+  // Generate m_numOfSubsets lists (= subsets of peersWhoCommitted)
   // If we have exactly the minimum num required for consensus, no point making
   // more than 1 subset
 
-  const unsigned int numSubsets = (peersWhoCommitted.size() < m_numForConsensus)
-                                      ? 1
-                                      : NUM_CONSENSUS_SUBSETS;
+  const unsigned int numSubsets =
+      (peersWhoCommitted.size() < m_numForConsensus) ? 1 : m_numOfSubsets;
```

Comment:
Now that I think about it, the check here should be `<=`, not `<`

Reply:
Agree.